beacon-chain/db/kv/key.go

```diff
@@ -51,3 +51,17 @@ func hasDenebBlindKey(enc []byte) bool {
 	}
 	return bytes.Equal(enc[:len(denebBlindKey)], denebBlindKey)
 }
+
+func hasElectraKey(enc []byte) bool {
+	if len(electraKey) >= len(enc) {
```

Comment:
We don't really need to do this, it would be fine to just call bytes.Equal.

Reply:
@kasey it is required or a runtime error for `slice bounds out of range` will happen when `len(enc)` is less than `len(electraKey)`

Reply:
You are thinking of bytes.Prefix, i think. It would be safe to call bytes.HasPrefix without this check

https://pkg.go.dev/bytes?utm_source=godoc#HasPrefix
```go
func hasElectraKey(enc []byte) bool {
	return bytes.hasPrefix(enc, electraKey)
}
```

Reply:
You're right, this is basically replicating `bytes.HasPrefix`.